lib/active_record/annotate/dumper.rb

```diff
@@ -4,8 +4,8 @@ module Dumper
       class << self
         def dump(table_name, connection = ActiveRecord::Base.connection, config = ActiveRecord::Base)
           string_io = StringIO.new
-          
-          if connection.table_exists?(table_name)
+
+          if connection.table_exists?(table_name) || connection.view_exists?(table_name)
```

Comment:
It seems like `#view_exists?` is a new method that doesn't exist in older versions of rails, therefore in older apps this will break.

Can you please add a check?
```suggestion
          if connection.table_exists?(table_name) ||
            connection.respond_to?(:view_exists?) && connection.view_exists?(table_name)
```